GVFS/GVFS/CommandLine/SparseVerb.cs

```diff
@@ -370,6 +408,8 @@ private void PruneFoldersOutsideSparse(ITracer tracer, Enlistment enlistment, Sp
                     this.ReportErrorAndExit(tracer, verbReturnCode, $"Failed to {PruneOptionName}. Exit Code: {verbReturnCode}");
                 }
             }
+
+            return directoriesToDehydrate;
         }
 
         private string[] GetDirectoriesOutsideSparse(string rootPath, SparseTable sparseTable)

```

Comment:
I know this isn't new to this change, but why are we using `string[]` instead of `List<string>`? In `GetDirectoriesOutsideSparse()`, we even use a `List<string>` internally but then `.ToArray()` it (which is slower, as it allocs a new array and copies all values to the new array).

Reply:
@derrickstolee that's a good question and I can't see why we'd need to use `string[]` either.  I will update these to `List<string>`.